expertise/create_dataset.py

```diff
@@ -6,27 +6,185 @@
 
 '''
 
-import os, json, argparse
+import json, argparse
 from datetime import datetime
+from pathlib import Path
+from itertools import chain
 
 import openreview
 from tqdm import tqdm
 
 from collections import defaultdict, OrderedDict
 
-def search_with_retries(client, term, max_retries=5):
-    results = []
-    for iteration in range(max_retries):
-        try:
-            results = openreview_client.search_notes(
-                term=term, content='authors', group='all',source='forum')
+from config import ModelConfig
```

Comment:
This needs to be changed to `from .config import ModelConfig` to make this work.

Reply:
Fixed!